game-headed/build.gradle

```diff
@@ -48,7 +48,7 @@ clean.doFirst {
 task downloadAssets {
     doLast {
         download {
-            src "https://github.com/triplea-game/assets/releases/download/10/game_headed_assets.zip"
+            src "https://github.com/triplea-game/assets/releases/download/18/game_headed_assets.zip"
```

Comment:
We really should establish release numbers for maps, this makes changes like this go so smoothly